src/app/components/message/MsgTypeRenderers.css.ts

```diff
@@ -0,0 +1,20 @@
+import { style } from '@vanilla-extract/css';
+
+export const buttonStyle = style({
+  width: '15%',
+  color: '#5e9ecf',
+  backgroundColor: 'transparent',
+  border: 'none',
+  padding: '0',
+  textDecoration: 'underline',
+  cursor: 'pointer',
+  transition: 'color 0.3s, text-decoration 0.3s',
```

Comment:
this is overriding the design system. could you please remove this style and use the Chip component if you want a smaller button with `fill="None"`?